packages/iota/src/models/payloads/IMilestonePayload.ts

```diff
@@ -24,37 +24,37 @@ export interface IMilestonePayload extends ITypeBase<7> {
     timestamp: number;
 
     /**
-     * The parents where this milestone attaches to.
+     * The timestamp of the milestone.
      */
-    parentMessageIds: string[];
+    lastMilestoneId: string;
 
     /**
-     * The merkle proof inclusions.
+     * The parents where this milestone attaches to.
      */
-    inclusionMerkleProof: string;
+    parentMessageIds: string[];
 
     /**
-     * The next PoW score.
+     * The merkle proof confirmation.
      */
-    nextPoWScore: number;
+    confirmedMerkleRoot: string;
 
     /**
-     * The milestone at which the next PoW score becomes active.
+     * The merkle proof applied.
```

Comment:
Also here description from TIP sounds better. i.e.
`The Merkle tree hash of all the messages applied by this milestone that contain a state-mutating transaction.`